chain/tendermint/proto/codec.proto

```diff
@@ -0,0 +1,280 @@
+syntax = "proto3";
+
+package fig.tendermint.codec.v1;
+
+option go_package = "github.com/figment-networks/tendermint-protobuf-def/codec";
+
+import "gogo.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
Do you know if these dependencies are needed?

Reply:
Just on this, `gogo.proto` is required, the `descriptor` isn't so it'll be removed